packages/core/src/framework/plugins/bridge/augury-bridge-request.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface AuguryBridgeRequest {
+  type: 'get_full_cd';
```

Comment:
Wondering if this string/enum could be extract into an isolated type?

Reply:
Ooops, I missed this one. I did the message type, but not the request type.